package.json

```diff
@@ -33,6 +33,7 @@
     "morgan": "^1.8.1"
   },
   "devDependencies": {
+    "@angular/cli": "^1.5.4",
```

Comment:
This dependency is probably unnecessary. Remove if possible, Angular is really large.